src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -289,7 +294,7 @@ export function updateLayeredMaterialNodeElevation(context, layer, node, force)
                 terrain.texture.flipY = false;
                 terrain.texture.needsUpdate = true;
             }
-            node.setTextureElevation(terrain);
+            if (terrain.texture.image.data[0] && terrain.texture.image.data[terrain.texture.image.data.length - 1] > -99999) node.setTextureElevation(terrain);
```

Comment:
Can you update the PR with the whitespaces fixes plus:
- a test of the 4 corners of the terrainData
- a comment stating that this is a compromise and that we should implement multi elevation layer instead
- make sure this works with texture images (see planar example)

With these changes this PR should be mergeable.